### PR TITLE
Update to go 1.15

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by roidelapluie** [comment]
@brian-brazil as maintainer, do you agree with the go.mod change? it means everyone needs to upgrade to go 1.15.

**Comment by SuperQ** [comment]
Actually, I'd prefer to avoid going right to 1.15 in go.mod. I'm just now getting the GitLab build pipeline up to 1.14. :crying_cat_face:

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus/prometheus
 
-go 1.14
+go 1.15
```
Comment (SuperQ) [inline go.mod:3]: Let's keep the minimum version 1.14 for now.

**Comment by roidelapluie** [comment]
back to 1.14, will merge after all is green, including build

**Pull request merged**